graphics/alembic/Portfile

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           active_variants 1.1
+
+github.setup        alembic alembic 1.7.12
+categories          graphics
+platforms           darwin
+license             BSD
+maintainers         @jasonliu-- openmaintainer
+homepage            https://www.alembic.io
+description         data representation scheme for storing computer \
+                    graphics scenes
+long_description    Alembic is a computer graphics interchange \
+                    framework, developed by Sony Pictures Imageworks \
+                    and Industrial Light & Magic. It distills complex, \
+                    animated scenes into a non-procedural, \
+                    application-independent set of baked geometric \
+                    results. It is very specifically not concerned \
+                    with storing portions of the assets such as \
+                    animation rigs. Instead, Alembic is used to bake \
+                    things like the geometry and physics of an \
+                    animated scene, the results of which are then \
+                    ready for hand-off to lighting and rendering.
+
+checksums           rmd160  edce9346991722242ac0060fccefaa5c7e25d788 \
+                    sha256  558559d3f5bb44976f4fb69a75caf87b8fa7cc8271db1b82d0e391495fd83971 \
+                    size    828000
+
+depends_lib-append  port:openexr
+
+# This is the default C++ standard for MacPorts,
+# so we don't need to specify it ourselves.
+# (Reference: https://trac.macports.org/wiki/CompilerSelection)
+#compiler.cxx_standard 1998
+
+configure.args      -DUSE_EXAMPLES=ON
```

Comment:
Please append here otherwise you are wiping out settings already supplied by the cmake portgroup